docs/ota-client-guide/modules/ROOT/pages/customise-targets-metadata.adoc

```diff
@@ -9,35 +9,45 @@ endif::[]
 
 include::partial$aktualizr-version.adoc[]
 
-In some cases, you might find it useful to include extra metadata about your software inside the signed Uptane metadata that OTA Connect delivers to your device. Some reasons you might want to do this include:
+You may need to include extra metadata about your software inside the signed Uptane metadata that OTA Connect delivers to your device. Some reasons you might want to do this include:
 
-* to provide installation instructions or scripts for an image that can't or shouldn't be included in the image itself
-* to add extra tracking fields for internal compliance or auditing
+* To provide installation instructions or scripts for an image that cannot or should not be included in the image itself.
+* To add extra tracking fields for internal compliance or auditing.
 
 To accommodate this use case, you can manually manage your Uptane metadata and add custom fields to your `targets.json` file.
 
-[NOTE]
-====
-For more information on additional use cases which will require customization of the Targets metadata, please refer to the section https://uptane.github.io/papers/ieee-isto-6100.1.0.0.uptane-standard.html#rfc.section.5.2.3.1.1['Custom metadata about images' in the Uptane standard].
-====
+NOTE: For more information on additional use cases that require customization of the targets metadata, see the https://uptane.github.io/papers/ieee-isto-6100.1.0.0.uptane-standard.html#rfc.section.5.2.3.1.1[related section] of the Uptane standard.
 
 == Prerequisites
 
-For this to be useful, you will need to write a custom handler for the metadata you add, so xref:libaktualizr-why-use.adoc[building your own update client using libaktualizr] is required. (Consult the https://advancedtelematic.github.io/aktualizr/index.html[Doxygen API reference] for how to access custom metadata in-client.)
+Before you write a custom handler for the metadata that you add:
+
+*  xref:libaktualizr-why-use.adoc[Build your own update client] using libaktualizr.
++
+To learn how to access custom metadata in-client, see the https://advancedtelematic.github.io/aktualizr/index.html[Doxygen API reference].
```

Comment:
"in-client" is a confusing phrase even for me. I know that isn't new to the text, but can we just say something like "in the embedded client"?